test_app/src/operator_indentation.erl

```diff
@@ -0,0 +1,86 @@
+-module(operator_indentation).
+
+a_large_predicate(PredOne, PredTwo, AValue) ->
+    something:very_long(based,
+                        on,
+                        PredOne,
+                        "1231231231231231313123123132sfsafsasfasdfasdfasdfsdf")
+        andalso
```

Comment:
This seems inconsistent... sometimes `andalso` is isolated, sometimes it's "after" the code, but in some (previous) modules we could also find it "before" the code.

Reply:
I think this one is _before_ running the formatter

Reply:
That is correct.

Reply:
Got it. So everything in `test_app` is _before_?

Reply:
`test_app/src` is _before_ … `test_app/after/src` is _after_  :)